docs/content/docs/crewai-crews/human-in-the-loop/flow.mdx

```diff
@@ -17,114 +17,100 @@ import InstallSDKSnippet from "@/snippets/install-python-sdk-crew.mdx";
-        import { Markdown } from "@copilotkit/react-ui"
-
-        function YourMainContent() {
-          // ...
-
-          useCopilotAction({
-            name: "confirmKickoffCrew",
-            available: "remote",
-            description: "Confirm the kickoff of the crew. You MUST call this tool before the crew can continue.",
-            parameters: [
-              { name: "jsonArguments", type: "string", description: "The JSON input to the crew", required: true },
-            ],
-            // [!code highlight:25]
-            renderAndWaitForResponse: ({ args, respond, status }) => {
-              return (
-                <div>
-                  <pre>{args.jsonArguments}</pre>
-
-                  <div className={`flex gap-4 pt-4 ${status !== "executing" ? "hidden" : ""}`}>
-                    <button
-                      onClick={() => respond?.("CANCEL")}
-                      disabled={status !== "executing"}
-                      className="border p-2 rounded-xl w-full"
-                    >
-                      Cancel
-                    </button>
-                    <button
-                      onClick={() => respond?.("KICKOFF")}
-                      disabled={status !== "executing"}
-                      className="bg-blue-500 text-white p-2 rounded-xl w-full"
-                    >
-                      Approve Kickoff
-                    </button>
-                  </div>
-                </div>
-              );
-            },
-          });
-
-          // ...
-        }
-        ```
-    </Step>
-
-    <Step>
-    ### Setup the CrewAI Agent
-    Now we'll setup the CrewAI agent. The actions defined on the frontend are made automatically available to the agent.
-
-    <Tabs groupId="language" items={["Python"]}>
-      <Tab value="Python">
-        ```python title="agent/sample_agent/agent.py"
-        sdk = CopilotKitRemoteEndpoint(
-            agents=[
-                CrewAIAgent(
-                    name="research_crew",
-                    description="Research agent",
-                    crew=ResearchCrew(),
-                ),
-            ],
-        )
-        ```
-      </Tab>
-    </Tabs>
-    </Step>
-    <Step>
-        ### Give it a try!
-        Try giving the crew enough information to complete the task, i.e. "Research the state of AI in 2025". You should see the custom UI component that allows you to approve or cancel the kickoff of the crew.
-    </Step>
-
+  <Step>
+    ### Run and connect your agent
+
+    Make sure your agent is running and connected to CopilotKit. If you haven’t set this up, start with the [Getting Started guide](/crewai-crews/quickstart/crewai).
+
+    You can also use our [coagent starter](https://github.com/CopilotKit/CopilotKit/tree/main/examples/coagents-starter-crewai-crews), which this guide builds upon.
+  </Step>
+
+  <Step>
+    ### Install the CopilotKit SDK
+    <InstallSDKSnippet components={props.components}/>
```

Comment:
I made a new snippet specifically for Crew - can we use that one here? Makes it so you don't have the typescript part